### DISCUSSION TITLE
GHGI table 3.13

### DISCUSSION THREAD

**Discussion by matthewlchambers** [USEPA/flowsa#304] [Q&A] [answered]
Has anyone incorporated table 3.13 into a GHGI FBS method yet? Someone (@bl-young ?) added it to the set of GHGI FBAs that are generated, but I'm having some questions as I try to incorporate it into my SEEA method, and I was wondering if anyone has worked with it yet.

**Comment by bl-young** [comment]
I started...but then held off as it was going to take some time to consider. 

E.g. https://github.com/USEPA/flowsa/commit/b3842f66cd04794264a64bf12ebe3a84e4342954 but then https://github.com/USEPA/flowsa/commit/094a4eed46e14ebae064994de127c89d2d4ed952

**Reply by matthewlchambers** [reply]
Gotcha. I'm taking a stab at it and would love feedback. I'll push it to a new branch when I'm satisfied it's at a reasonable point.

**Reply by bl-young** [reply]
Great, the ghgi branch I assume?
And this would be for https://github.com/USEPA/GHGbyindustrydocs/issues/30 and https://github.com/USEPA/GHGbyindustrydocs/issues/36 ?

Great I will focus on other things rather than that table for now then.

**Comment by bl-young** [comment, marked as answer]
@matthewlchambers note that I went ahead and made this update on the core GHGI method when I updated it to the recursive method:
https://github.com/USEPA/flowsa/pull/332
